src/ps33.js

```diff
@@ -0,0 +1,91 @@
+const auth = React.useMemo(
+    () => ({
+      login: async (email, password) => {
+        const { data } = await axios.post(`${BASE_URL}/auth/local`, {
+          identifier: email,
+          password,
+        });
+        const user = {
+          email: data.user.email,
+          token: data.jwt,
+          id: data.user.id,
+        };
+        dispatch(createAction('SET_USER', user));
```

Comment:
no magic strings, please export action strings to enum, to have one source of truth. This will allow to change naming or check where it's being used easier.